src/main/java/cz/smarteon/loxone/app/state/events/LockedEvent.java

```diff
@@ -0,0 +1,17 @@
+package cz.smarteon.loxone.app.state.events;
+
+import cz.smarteon.loxone.app.state.Locked;
+import lombok.Builder;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder
+@Jacksonized
+public class LockedEvent {
+
+    @Builder.Default
+    Locked locked = Locked.NO;
```

Comment:
not sure how the lombok works here, but I guess it won't make the fields private right? So let's make the private please

Reply:
Lombok makes them private, so private modifier is not needed